pombola/south_africa/views.py

```diff
@@ -1962,3 +1962,101 @@ def get_context_data(self, **kwargs):
             context['data'] = None
 
         return context
+
+class SAMpAttendanceView(TemplateView):
```

Comment:
I think the pages served from this view could be cached safely - perhaps using cache_control to set the max age to 10 minutes would be worthwhile?